prediction_market_agent/agents/autogen_general_agent/prompts.py

```diff
@@ -0,0 +1,22 @@
+INFLUENCER_PROMPT = """
+        You are a Public Relations employee that is the public face of an AI agent that places bets
```

Comment:
You could make this part of the prompt first-person (i.e. `You are an AI agent that places bets...`) and remove `Formulate the tweet in the first person.` below. As you prefer.

Reply:
Yes, that sounds cleaner. Implemented.